packages/kubernetes-1.22/etc-kubernetes-pki.mount

```diff
@@ -8,9 +8,9 @@ Wants=selinux-policy-files.service
 
 [Mount]
 What=tmpfs
-Where=/etc/kubernetes/pki
+Where=/etc/kubernetes/pki/private
 Type=tmpfs
 Options=nosuid,nodev,noexec,noatime,context=system_u:object_r:secret_t:s0,mode=0700
 
 [Install]
-WantedBy=preconfigured.target
\ No newline at end of file
+WantedBy=preconfigured.target
```

Comment:
Do the added newlines here and in the mount file under the 1.21 directory matter?

Reply:
They shouldn't functionally matter. They were added here as there was some guidance to configured IDE settings to always have a newline at the end of files when editing. I've gotten a few review comments when my updates were missing that. So including them in this change as a trivial update.

Reply:
Nice! Makes sense! Wasn't sure about `.mount` files